CMakeLists.txt

```diff
@@ -11,11 +11,16 @@ find_package(everest-cmake 0.1 REQUIRED
 )
 
 # options
-option(BUILD_TESTING "Run unit tests" OFF)
+option(${PROJECT_NAME}_BUILD_TESTING "Build unit tests, used if included as dependency" OFF)
```

Comment:
I am not so sure what the idea is behind the 2 types of `BUILD_TESTING` flags but `LIBOCPP_BUILD_TESTING` seems to work for me :+1: